builtin/settingtypes.txt

```diff
@@ -1915,16 +1915,18 @@ emergequeue_limit_diskonly (Limit of emerge queues on disk) int 64
 emergequeue_limit_generate (Limit of emerge queues to generate) int 64
 
 #    Number of emerge threads to use.
+#    WARNING: Currently there are multiple bugs that cause crashes when
+#    'num_emerge_threads' is larger than 1. Until this warning is removed it is
+#    strongly recommended to set this value to '1'.
```

Comment:
since the default is 1, this should say `change this value to any value > 1`

Reply:
Will reword.

Reply:
Done.